R/Widget_ScatterPlot.R

```diff
@@ -48,6 +48,7 @@ Widget_ScatterPlot <- function(
     dfBounds = dfBounds,
     bAddGroupSelect = bAddGroupSelect,
     strShinyGroupSelectID = strShinyGroupSelectID,
+    strFootnote = ifelse(bAddGroupSelect, "Size of the point is relative to size of the group.", ""),
```

Comment:
@samussiah in no universe am I a JS writer, but this seemed simple enough to take as an initial task, is the logical `bAddGroupSelect` the if condition you meant in #1728 ?

Reply:
You're on the right track but the logic should be around `dfGroups` instead of `bAddGroupSelect`.